pandas/tests/io/pytables/common.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+import os
+import tempfile
+from contextlib import contextmanager
+from distutils.version import LooseVersion
+
+import pandas.util.testing as tm
+
+
+tables = pytest.importorskip('tables')
+from pandas.io import pytables as pytables  # noqa:E402
+from pandas.io.pytables import (TableIterator,  # noqa:E402
+                                HDFStore, get_store, Term, read_hdf,
+                                PossibleDataLossError, ClosedFileError)
+
+
+_default_compressor = ('blosc' if LooseVersion(tables.__version__) >=
+                       LooseVersion('2.2') else 'zlib')
+
+
+# contextmanager to ensure the file cleanup
+
+
+def safe_remove(path):
```

Comment:
would like to tun these into fixtures, also this removes the need for the Base class entirely. ok with doing this in another pass though